apps/financial-aid/api/src/app/modules/nationalRegistryXRoadModule/nationalRegistryXRoad.resolver.ts

```diff
@@ -0,0 +1,51 @@
+import { UseGuards } from '@nestjs/common'
+import { Resolver, Query, ResolveField, Parent, Context } from '@nestjs/graphql'
+import type { User } from '@island.is/auth-nest-tools'
+import {
+  CurrentUser,
+  IdsAuthGuard,
+  IdsUserGuard,
+  Scopes,
+  ScopesGuard,
+} from '@island.is/auth-nest-tools'
+import { Audit } from '@island.is/nest/audit'
+import { NationalRegistryXRoadService } from '@island.is/api/domains/national-registry-x-road'
+import { Person } from './models/person.model'
+import { UserSpouse } from './models/userSpouse.model'
+
+@UseGuards(IdsAuthGuard, IdsUserGuard, ScopesGuard)
+@Scopes('@skra.is/individuals')
```

Comment:
You should use your GQL api scope `@samband.is/internal` here as the skra.is scope is intended for skra.is resources. And here you are protecting your gql query.